docs/source/tutorial/add_another_pipeline.md

```diff
@@ -0,0 +1,532 @@
+# Create a data science pipeline
+
+This section explains the following:
+
+* How to add a second Kedro pipeline for data science code that extends the default project pipeline
+* How to 'slice' the project and run just part of the default pipeline
+* How to visualise the default project pipeline
+* (Optional) How to specify the way the pipeline nodes are run: sequentially or in parallel
+* (Optional) How to make a modular pipeline
```

Comment:
I'd link to the modular pipeline page here and find some way of emphasising that this is one of the standout features of Kedro that allows you to reuse logic easily